


lambda: 0.1
max_dist_laser: 5

static_filter: true
Static_map_removal_tolerance: 4

polygon_tolerance: 1.02 # 1.04
polygon_side_min_points_required: 4

min_twist_detection: 0.3 #0.08

similarty_side_length_weight: 4.0
similarty_side_angle_weigh: 2.0
similarty_side_amount_weight: 0
similarty_track_xposition_weight: 5.2
similarty_track_yposition_weight: 5.2
similarty_previous_position_weight: 2.2

max_similarty_deviation: 1.5

min_size_cluster: 10


TRACKER_LIFE: 100
CONFIRMED_TRACK: 20

mapframeid: map
odomframeid: odom_chassis
base_laser_frame: base_laser
base_frame: base_link